venues/aclweb.org/ACL/reviewer_ac_groups.py

```diff
@@ -0,0 +1,130 @@
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+sac_name_dictionary = tracks.sac_name_dictionary
+
+
+# Create Ethics AC group 
+ethics = client.post_group(openreview.Group(
+    id = 'aclweb.org/ACL/2022/Conference/Ethics_Area_Chairs',
+    signatures = [
+        'aclweb.org/ACL/2022/Conference'
+        ],
+    signatories=[
+        'aclweb.org/ACL/2022/Conference'
+        ],
+    readers = [
+        'aclweb.org/ACL/2022/Conference',
+        'aclweb.org/ACL/2022/Conference/Ethics_Area_Chairs'
+    ],
+    writers = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        members = [
+            '~Christiane_Fellbaum1', 
+            'blodgett@cs.umass.edu'
+        ]
+))
+# Create Ethics Reviewers Group 
+ethics_reviewers = client.post_group(openreview.Group(
+        id = f'aclweb.org/ACL/2022/Conference/Ethics_Reviewers',
+        signatures = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        signatories=[
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        readers = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        writers = [
+            'aclweb.org/ACL/2022/Conference'
+             ]
+        ))
+# For submission in submissions, add paperx/Reviewers group and AC group as readers 
+# Currently assumes submissions_list is a list of Notes 
+submissions_forum_list = ['_SmerUlTll0']
+# For Each submission, create reviewer group, add reviewer group and AC group to readers 
+for submission_forum in tqdm(submissions_forum_list): 
+    submission = client.get_note(submission_forum)
+    paper_track = sac_name_dictionary[submission.content["track"]]
+    track_sac_id = f'aclweb.org/ACL/2022/Conference/{paper_track}/Senior_Area_Chairs'
+    conflict_id = f'aclweb.org/ACL/2022/Conference/Paper{submission.number}/Conflicts'
+    # Create Ethics Reviewer group 
+    ethics_reviewers_paper = client.post_group(openreview.Group(
+        id = f'aclweb.org/ACL/2022/Conference/Paper{submission.number}/Ethics_Reviewers',
+        signatures = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        signatories=[
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        readers = [
+            'aclweb.org/ACL/2022/Conference',
+            'aclweb.org/ACL/2022/Conference/Ethics_Area_Chairs',
+            f'aclweb.org/ACL/2022/Conference/Paper{submission.number}/Ethics_Reviewers'
+            ],
+        writers = [
+            'aclweb.org/ACL/2022/Conference'
+             ],
+        nonreaders= submission.nonreaders,
+        anonids= True
+        ))
+    client.add_members_to_group('aclweb.org/ACL/2022/Conference/Ethics_Reviewers', ethics_reviewers_paper.id)
```

Comment:
no need to do this, I'm adding the assigned reviewer to this group after the assignment is done.